package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "restaurant-page-JS",
+  "version": "1.0.0",
+  "description": "This is a project from [the Odin project](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) for implementation of webpack and continued DOM manipulation in Javascript",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
- [ ] We can safely delete the `test` script since no tests are going to be implemented.